ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/LoadedPaywallComponents.kt

```diff
@@ -0,0 +1,83 @@
+@file:JvmSynthetic
+
+package com.revenuecat.purchases.ui.revenuecatui.components
+
+import android.os.LocaleList
+import androidx.compose.material3.adaptive.currentWindowAdaptiveInfo
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.window.core.layout.WindowSizeClass
+import com.revenuecat.purchases.paywalls.components.common.LocaleId
+import com.revenuecat.purchases.ui.revenuecatui.components.modifier.background
+import com.revenuecat.purchases.ui.revenuecatui.components.properties.toBackgroundStyle
+import com.revenuecat.purchases.ui.revenuecatui.components.state.PackageContext
+import com.revenuecat.purchases.ui.revenuecatui.components.style.StyleFactory
+import com.revenuecat.purchases.ui.revenuecatui.data.PaywallState
+import com.revenuecat.purchases.ui.revenuecatui.data.processed.VariableDataProvider
+import com.revenuecat.purchases.ui.revenuecatui.helpers.getOrThrow
+import com.revenuecat.purchases.ui.revenuecatui.helpers.toResourceProvider
+import java.util.Locale
+
+@Composable
```

Comment:
Should we mark it as `@JvmSynthetic`?

Reply:
I tried to see if it's possible to call Composable functions from Java, but can't seem to figure out. So I guess it's not possible? Maybe the `@file:JvmSynthetic` at the top of the file is enough?